docs/GettingStarted.md

```diff
@@ -64,7 +64,8 @@ of the Datadog tracer, you can override the following defaults:
       tracer: Datadog.tracer,
       debug: false,
       trace_agent_hostname: 'localhost',
-      trace_agent_port: 8126
+      trace_agent_port: 8126,
+      env: Rails.env
```

Comment:
It looks like this change has yielded unexpected results, at least for me. With this change, some spans from the app—and _only some_ but not all, oddly—are getting `env:production` set.

I have a Rails application that we run in multiple named production-like environments, e.g. `staging` and `production`. Both systems' `Rails.env` is `production` per Rails conventions. I differentiate between them by setting the `env:` tag in the agent's `datadog.conf`.

At a minimum, I'd call this change backwards-incompatible: upgrading the gem results in data getting sent with an unexpected `env:` tag.

Was this change to handle deployments where a single Datadog agent on a host would be receiving and forwarding data from multiple applications (not just development/test/production) and so therefore the application needs a way to name its stuff (traces? spans? I'm new here.).

I'm new to APMing with Datadog, so I fully admit I could be doing something wrong or could be doing something better.